src/components/sbb-card/sbb-card.scss

```diff
@@ -73,35 +73,38 @@
   &:focus-visible {
     @include focusOutline;
   }
-}
 
-:host([active]) {
-  .sbb-card::before {
-    border: var(--sbb-border-width-2x) solid var(--sbb-color-smoke-default);
+  :host([href]) & {
+    display: block;
+    color: inherit;
+    text-decoration: none;
   }
-}
 
-.sbb-card__link {
-  display: block;
-  color: inherit;
-  text-decoration: none;
-}
+  // stylelint-disable-next-line plugin/stylelint-bem-namics
+  :host(:not([href])) & {
+    @include buttonReset;
+    @include text-inherit;
 
-.sbb-card__button {
-  @include buttonReset;
-  @include text-inherit;
+    cursor: pointer;
+  }
 
-  cursor: pointer;
+  // stylelint-disable plugin/stylelint-bem-namics
+  :host([href]) &,
+  :host(:not([href])) & {
```

Comment:
This is always either of the two cases. Would it make sense to just move these rules directly to .sbb-card?

Reply:
It was not working due `buttonReset`, rules moved to .sbb-card and added a note in :not([href])